Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This file is automatically generated by Jina executor normalizer plugin.
```

Comment:
I think you should remove the dockerfile as it is automatically created.

Reply:
it doesn't work. See thread in slack